modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -23,6 +23,7 @@ The workspace engine and authentication system used until {prod-prev-short} vers
 [source,bash,subs="+attributes"]
 ----
 export INSTALLATION_NAMESPACE={prod-namespace} #<1>
+export OPERATOR_NAMESPACE={prod-namespace}     #<1>
```

Comment:
does this default to a different value than `INSTALLATION_NAMESPACE` ? Does it need a different footnote number?

Reply:
1. Migration from `latest` channel: OPERATOR_NAMESPACE = INSTALLATION_NAMESPACE
2. Migration from`tech-preview-channel` OPERATOR_NAMESPACE != INSTALLATION_NAMESPACE  
So, we need to differentiate these 2 namespaces